core/build.gradle

```diff
@@ -1,4 +1,6 @@
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android-extensions'
```

Comment:
Why was `kotlin-android-extensions` included here?

Reply:
It was auto generated when I added Kotlin. I'll remove it 😅

Reply:
LOL.